tests/storm/locustfile.py

```diff
@@ -0,0 +1,67 @@
+import json
+from locust import HttpUser, between, task
+
+from tests.storm.common import execute_async
+
+
+class RookUser(HttpUser):
+    host = "http://localhost:5000"
+    wait_time = between(1, 10)
+
+    @task(4)
```

Comment:
(4) means 4 concurrent requests?

Reply:
I'm also new to locust. The number in `task(4)` means that is 4 times more likely to be called:

Reply:
https://docs.locust.io/en/stable/quickstart.html